src/Sulu/Bundle/WebsiteBundle/Tests/Unit/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProviderTest.php

```diff
@@ -680,6 +680,53 @@ public function testGetCollectionTrailingSlash()
         $this->assertEquals('/de/qwertz', $route->getDefaults()['url']);
     }
 
+    public function testGetCollectionTrailingSlashWithQueryParams()
+    {
+        $attributes = $this->prophesize(RequestAttributes::class);
+
+        $portal = new Portal();
+        $portal->setKey('portal');
+        $webspace = new Webspace();
+        $webspace->setKey('webspace');
+        $webspace->setTheme('theme');
+        $portal->setWebspace($webspace);
+        $attributes->getAttribute('portal', null)->willReturn($portal);
+        $attributes->getAttribute('localization', null)->willReturn(new Localization('de'));
+        $attributes->getAttribute('matchType', null)->willReturn(RequestAnalyzer::MATCH_TYPE_FULL);
+        $attributes->getAttribute('resourceLocator', null)->willReturn('/foo/');
+        $attributes->getAttribute('resourceLocatorPrefix', null)->willReturn('/de');
+
+        $this->resourceLocatorStrategy->loadByResourceLocator('/foo', 'webspace', 'de')->willReturn('some-uuid');
+
+        $redirectTargetDocument = $this->prophesize(ResourceSegmentBehavior::class);
+        $redirectTargetDocument->getResourceSegment()->willReturn('/foo');
+
+        $document = $this->prophesize(TitleBehavior::class)
+                         ->willImplement(RedirectTypeBehavior::class)
+                         ->willImplement(StructureBehavior::class)
+                         ->willImplement(UuidBehavior::class);
```

Comment:
We don't align items like that, because that means we have change the alignment also when the variable name above changes. So please indent the above three lines only with 4 spaces.

Reply:
Fixed. Sorry about that. 
Needless to say, PHPStorm does that automatically. 

Unfortunately, I couldn't find yet a way to import the .styleci.yml into my IDE. Perhaps if you could also post a sample Idea XML or JSCS code style file to the contribution guidelines, then we wouldn't run into issues like that.

Reply:
You are completely right. I would also be in favor of using PHP-CS-Fixer directly, instead of relying on StyleCI, then everybody could have the checks locally as well :confused: /cc @alexander-schranz That's something that could be done on hacktoberfest, right? :smile: